### ISSUE TITLE
tests fail

### ISSUE THREAD

**Issue by roryk** [AndreMacedo88/VEnCode#6]
Hi everyone,

None of the test seem to be working-- are they meant to be runnable? It looks like they require files that aren't available in the repository. Is there a way an example dataset could be included with the tests? It looks like it is intended to be included but didn't get pushed to the repository.

**Comment by AndreMacedo88** [comment]
Hi Rory, thank you very much for your help with the module the installation fixes and for your interest in this project!
About the issue, I'm so sorry, indeed I didn't push the test files to the repository! I just finished doing just that, so the tests should be runnable after following the deployment instructions. I also updated the readme file to clarify the simplest way to run the tests, which is probably by using the unittest module.
Finally, I updated the deployment instructions. Due to early path choices, the default path for both work and testing data must be the local repository's parent directory, so the directory "Files" included in this repository must be moved up to the parent dir in your local machine for the tests to work. And if you download the FANTOM5 data to use this module, that data must be placed in the same directory "Files" outside the repository. It's not ideal and I must change that in the near future.
